### ISSUE TITLE
Maps

### ISSUE THREAD

**Issue by morkor** [CLARIAH/wp4-requirements#138]
bring maps back using LOD.

Question: should I do something static until then? No.

**Comment by rlzijdeman** [comment]
nope, the non-LD approach for the data was already a shortcut and I expect the maps soonishly.

**Comment by morkor** [comment]
answer is no for non-LOD but why close the generic map issue?
